tasks/util/validateTaskPlan.test.js

```diff
@@ -0,0 +1,36 @@
+import test from 'ava'
+import jsf from 'json-schema-faker'
+
+const Ajv = require('ajv')
+const { validateSchema } = require('feathers-hooks-common')
+const schema = require('../schemas/taskPlan')
```

Comment:
it seems the library expects to find referenced schemas at a particular remote URL, according to the JSON schema standard. we want to somehow locally pass in all the schemas up front and tell it not to look remotely.